### ISSUE TITLE
Decrease caching time on Dataset list page?

### ISSUE THREAD

**Issue by flaneuse** [biothings/discovery-app#132]
A user mentioned that after registering a dataset using the NIAID Guide, when he went to look at the [registered datasets](https://discovery.biothings.io/dataset?guide=/guide/niaid), the number hadn't updated with his new dataset on Chrome, even a day later.  The new dataset was shown when he viewed the page in Edge.  A hard refresh (ctrl-shift-R) in Chrome solved the problem, so I assume the API call to get the dataset list is being cached for days.

**Comment by newgene** [comment]
yeah, default is max 7-day cache, we can reduce it, or even remove the cache.
